.github/workflows/sklearn-test.yml

```diff
@@ -0,0 +1,62 @@
+name: sklearn-test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  get-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - name: Install MLflow
+        run: |
+          pip install -e .
+      - id: set-matrix
+        run: |
+          MIN_VERSION=$(python -c "import mlflow; print(mlflow.sklearn.MIN_VERSION)")
+          MAX_VERSION=$(python -c "import mlflow; print(mlflow.sklearn.MAX_VERSION)")
+          VERSIONS_JSON=$(python get_versions.py --package-name scikit-learn --min-ver $MIN_VERSION --max-ver $MAX_VERSION)
+          echo "::set-output name=matrix::$VERSIONS_JSON"
```

Comment:
Extract `MIN_VERSION` and `MAX_VERSION`, and construct a version matrix from them.